tsl/test/isolation/expected/cagg_insert.out

```diff
@@ -1,141 +1,168 @@
+step Refresh3: CALL refresh_continuous_aggregate('continuous_view_2', NULL, 15); <waiting ...>
+step I2c: COMMIT;
+step UnLockCagg2: ROLLBACK;
+step Refresh3: <... completed>
+
+starting permutation: I2b LockCagg2 Refresh3 Refresh I21 I2c UnLockCagg2
+step I2b: BEGIN; SET LOCAL lock_timeout = '500ms'; SET LOCAL deadlock_timeout = '10ms';
+step LockCagg2: BEGIN; SELECT lock_cagg('continuous_view_2');
+lock_cagg
+---------
+         
+(1 row)
+
+step Refresh3: CALL refresh_continuous_aggregate('continuous_view_2', NULL, 15); <waiting ...>
+step Refresh: CALL refresh_continuous_aggregate('continuous_view_1', NULL, 15);
+step I21: INSERT INTO ts_continuous_test_2 VALUES (1, 1);
+step I2c: COMMIT;
+step UnLockCagg2: ROLLBACK;
+step Refresh3: <... completed>
+
+starting permutation: Sb LockCagg1 Refresh S1 Sc UnLockCagg1
 step Sb: BEGIN; SET LOCAL lock_timeout = '500ms'; SET LOCAL deadlock_timeout = '10ms';
-step LockCagg: BEGIN; SELECT lock_cagg('continuous_view');
+step LockCagg1: BEGIN; SELECT lock_cagg('continuous_view_1');
 lock_cagg
 ---------
          
 (1 row)
 
-step Refresh: CALL refresh_continuous_aggregate('continuous_view', NULL, 15); <waiting ...>
-step S1: SELECT count(*) FROM ts_continuous_test;
+step Refresh: CALL refresh_continuous_aggregate('continuous_view_1', NULL, 15); <waiting ...>
+step S1: SELECT count(*) FROM ts_continuous_test_1;
 count
 -----
    30
 (1 row)
 
 step Sc: COMMIT;
-step UnlockCagg: ROLLBACK;
+step UnLockCagg1: ROLLBACK;
 step Refresh: <... completed>
 
-starting permutation: Sb LockCagg S1 Refresh Sc UnlockCagg
+starting permutation: Sb LockCagg1 S1 Refresh Sc UnLockCagg1
 step Sb: BEGIN; SET LOCAL lock_timeout = '500ms'; SET LOCAL deadlock_timeout = '10ms';
-step LockCagg: BEGIN; SELECT lock_cagg('continuous_view');
+step LockCagg1: BEGIN; SELECT lock_cagg('continuous_view_1');
 lock_cagg
 ---------
          
 (1 row)
 
-step S1: SELECT count(*) FROM ts_continuous_test;
+step S1: SELECT count(*) FROM ts_continuous_test_1;
 count
 -----
    30
 (1 row)
 
-step Refresh: CALL refresh_continuous_aggregate('continuous_view', NULL, 15); <waiting ...>
+step Refresh: CALL refresh_continuous_aggregate('continuous_view_1', NULL, 15); <waiting ...>
 step Sc: COMMIT;
-step UnlockCagg: ROLLBACK;
+step UnLockCagg1: ROLLBACK;
 step Refresh: <... completed>
 
 starting permutation: Ib LockInvalThr Refresh I1 Ic UnlockInvalThr
 step Ib: BEGIN; SET LOCAL lock_timeout = '500ms'; SET LOCAL deadlock_timeout = '10ms';
 step LockInvalThr: BEGIN; LOCK TABLE _timescaledb_catalog.continuous_aggs_invalidation_threshold IN SHARE MODE;
-step Refresh: CALL refresh_continuous_aggregate('continuous_view', NULL, 15); <waiting ...>
-step I1: INSERT INTO ts_continuous_test VALUES (1, 1);
-step Ic: COMMIT; <waiting ...>
+step Refresh: CALL refresh_continuous_aggregate('continuous_view_1', NULL, 15); <waiting ...>
+step I1: INSERT INTO ts_continuous_test_1 VALUES (1, 1);
+step Ic: COMMIT;
```

Comment:
Since commit is not waiting here, are you actually testing for any race?

Reply:
Now we're not locking the invalidation_threshold table anymore so it is expected to don't wait (need to fix some comments in the test spec)